src/busco/script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+
+[[ "$par_tar" == "false" ]] && unset par_tar
+[[ "$par_force" == "false" ]] && unset par_force
+[[ "$par_offline" == "false" ]] && unset par_offline
+[[ "$par_opt_out_run_stats" == "false" ]] && unset par_opt_out_run_stats
+[[ "$par_quiet" == "false" ]] && unset par_quiet
+[[ "$par_restart" == "false" ]] && unset par_restart
+[[ "$par_auto_lineage" == "false" ]] && unset par_auto_lineage
+[[ "$par_auto_lineage_euk" == "false" ]] && unset par_auto_lineage_euk
+[[ "$par_auto_lineage_prok" == "false" ]] && unset par_auto_lineage_prok
+[[ "$par_augustus" == "false" ]] && unset par_augustus
+[[ "$par_long" == "false" ]] && unset par_long
+[[ "$par_scaffold_composition" == "false" ]] && unset par_scaffold_composition
+[[ "$par_miniprot" == "false" ]] && unset par_miniprot
+
+if [[ -n "$par_output_prefix" ]]; then 
+    prefix="$par_output_prefix"
+else 
+    prefix="$(basename -- $par_input)"
+fi
+
+if [[ -n "$meta_cpus" ]]; then 
+    cpus="$meta_cpus"
+else 
+    cpus=1
+fi
+
+busco \
+    --in "$par_input" \
+    --mode "$par_mode" \
+    --cpu "$cpus" \
```

Comment:
You could also use

```suggestion
    ${meta_cpus:+--cpu "${meta_cpus}" \

Reply:
This gave me trouble - if meta_cpu's isn't defined, null value gets passed to busco --cpu, that's why I explicitly set it to 1 instead